application/backup.sh

```diff
@@ -1,4 +1,29 @@
 #!/usr/bin/env sh
+# Error Tracking
+if [ ! -z "${ SENTRY_DSN }"]; then
+    eval "$(curl -sL https://sentry.io/get-curl.sh)"
+    SENTRY_TRACE_ID=$(uuidgen)
```

Comment:
I think `uuidgen` isn't installed in the ubuntu Docker image. Maybe the trace ID isn't necessary. What reference did you find that described it?

Reply:
> I think `uuidgen` isn't installed in the ubuntu Docker image. Maybe the trace ID isn't necessary. What reference did you find that described it?

this is something i missed. i have installed in my local, and i missed to consider Ubuntu docker image. As for the trace ID just wanted to try tracing . its not that much necessary one here as we have the name of the db, status and hostname to navigate.

Reply:
> > I think `uuidgen` isn't installed in the ubuntu Docker image. Maybe the trace ID isn't necessary. What reference did you find that described it?
> 
> this is something i missed. i have installed in my local, and i missed to consider Ubuntu docker image. As for the trace ID just wanted to try tracing . its not that much necessary one here as we have the name of the db, status and hostname to navigate.

i'll this block to keep it simple.